akka-actor/src/main/scala-2.13/akka/util/ByteString.scala

```diff
@@ -273,7 +275,7 @@ object ByteString {
 
   /** INTERNAL API: ByteString backed by exactly one array, with start / end markers */
   private[akka] object ByteString1 extends Companion {
-    val empty: ByteString1 = new ByteString1(Array.empty[Byte])
+    val empty: ByteString1 = new ByteString1(new Array[Byte](0), 0, 0)
```

Comment:
Not that it matters here but `Array.empty` is quite complicated. Removed the aux constructor below that was only used here.

Reply:
There is `Array#emptyByteArray` though, which is pretty much the same as this

Reply:
Great, thanks, TIL